src/plugins/comicsPlayer/plugin.js

```diff
@@ -5,6 +5,9 @@ import dialogHelper from '../../components/dialogHelper/dialogHelper';
 import keyboardnavigation from '../../scripts/keyboardNavigation';
 import { appRouter } from '../../components/appRouter';
 import ServerConnections from '../../components/ServerConnections';
+// eslint-disable-next-line import/named, import/namespace
```

Comment:
Is this for `No Babel config ...` from ESLint? If so, it will be fixed in my ES6 PR.

Reply:
Probably. I just copied it from the image slideshow import.